Assign graph to cpu in example/udacity/5_word2vec.

jyegerlehner opened the pull request:
Here's what I think is a fix for [1297](https://github.com/tensorflow/tensorflow/issues/1297). Please see the issue for discussion.

If there's a way to do this without all the whitespace changes please let me know. I'm new to Python. The only real change is addition of the `with tf.device():` statement.

tensorflow-jenkins commented:
Can one of the admins verify this patch?

jyegerlehner commented:
Signed CLA.

vincentvanhoucke commented on tensorflow/examples/udacity/5_word2vec.ipynb:
```diff
@@ -422,38 +422,44 @@
         "graph = tf.Graph()\n",
         "\n",
         "with graph.as_default():\n",
```
Thanks for doing this. May I ask you to do it on one line instead:
  with graph.as_default(), tf.device('/cpu:0'):
that will remove all the space changes.

jyegerlehner committed:
Assign graph to cpu in example/udacity/assignment5.

This was required so the code will execute on tensorflow
installations with gpu support. SparseApplyAdagrad op has
no gpu implementation at this time and causes exception to be
thrown.

jyegerlehner commented:
> May I ask you to do it on one line instead:

Sure thing Dr. V. There you go.
